requirements.txt

```diff
@@ -0,0 +1,7 @@
+tensorflow>=2.1.0
+tensorflow-gpu>=2.1.0
+numpy>=1.19.1
+tqdm>=4.48.2
+opencv>=4.4.0
```

Comment:
Maybe anaconda can work properly.
https://github.com/zzh8829/yolov3-tf2/blob/master/requirements-gpu.txt
The original project also has it (opencv) in requirements.txt file.

Reply:
Anaconda is evil (in my opinion). This could be listed explicitly on the webpage as an external dependency. The reason is that users are most likely run `pip install -r requirment.txt` and fails.

Reply:
Does opencv have any alias? Is there any other way to add opencv in requirements.txt file?

Reply:
This is not a codebase solely for training yolo, and OpenCV is in fact recommended to install from source due to its level of customization. Still don't think this should be a generic **python** dependency.

Reply:
Cool!